package.json

```diff
@@ -42,6 +42,7 @@
   "devDependencies": {
     "@babel/types": "^7.24.0",
     "@eslint-types/typescript-eslint": "^7.0.2",
+    "@polka/compression": "^1.0.0-next.25",
```

Comment:
I think this should be in `packages/vite/package.json`

Reply:
Oops, my bad. I fixed that.